src/ec/packages/ec-component-text-input/__snapshots__/text-input.test.js.snap

```diff
@@ -0,0 +1,42 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`EC - Text input Default renders correctly 1`] = `
+"<div class=\\"ecl-form-group ecl-form-group--text-input\\">
+  <label class=\\"ecl-form-label\\" for=\\"example-input-id-1\\"> Label </label>
+
+  <input
+    id=\\"example-input-id-1\\"
+    name=\\"\\"
+    type=\\"text\\"
+    placeholder=\\"Placeholder\\"
+    class=\\"ecl-text-input\\"
+  />
+</div>
+"
+`;
+
+exports[`EC - Text input Disabled renders correctly 1`] = `
+"<div class=\\"ecl-form-group ecl-form-group--text-input\\">
+  <input
```

Comment:
This is not a disabled input. Please make sure the mapping between `import dataDisabled from '@ecl/ec-specs-text-input/demo/data--disabled'` and what the component consumes matches